### PR TITLE
Components

### PR DESCRIPTION
**Pull request by lengushuru**
**In this pull request the following changes were made**

- Deleted all the boilerplate from Create React App (text, images, styles).
- Created a new directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, created a React component that matches provide design
- Implemented the design only (JSX and CSS)
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Used functional components
- Created at least 1 child component
- Once the Calculator is ready, imported it and use it in top-level component: App.js
- Tested  it in the browser; it rendered without problems.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
```
Comment (Zuhaib042) [inline src/App.js:2]: - [ ] You may want to delete this line as deleting all the boilerplate generated with CRA must be removed.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+const Calculator = () => (
```
Comment (Zuhaib042) [inline src/components/Calculator.js:1]: - [ ] This component should have at least one child component which you would import here in this file and use it in your Calculator component as it is the project requirement.
  - Create at least 1 child component

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @lengushuru  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

## Highlights
✅ Professional PR
✅ Professional Readme

## Required Changes ♻️
- [ ] Kindly consider removing the `App.css` file as it was generated by the CRA command and serves no purpose now.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (lengushuru) [reply src/components/Calculator.js:1]: Okay 👍

File: src/components/Calculator.js [hunk 3]
```diff
@@ -1,11 +1,17 @@
+import PropTypes from 'prop-types';
+
+const Display = ({ output }) => (
+  <div className="display">{ output }</div>
+);
```
Comment (PinkMoon25) [inline src/components/Calculator.js:5]: - [ ] good job making the changes requested, however as mentioned by the previous reviewer https://github.com/lengushuru/math-magician/pull/2#discussion_r1143899728 , please notice that this should be a component itself and should be in a separate file just like Calculator.js, then you would import this component here using the import statement and use it in your Calculator component, having to different Components in the same file does not make much sense, please fix it :heavy_check_mark:

File: src/components/Calculator.js [hunk 4]
```diff
@@ -47,4 +53,8 @@ const Calculator = () => (
   </div>
 );
 
+Display.propTypes = {
+  output: PropTypes.string.isRequired,
+};
+
```
Comment (PinkMoon25) [inline src/components/Calculator.js:59]: - [ ] [OPTIONAL] The proptype check is failing as currently you are passing number and not a string which you have mentioned here, please fix it :+1:

**Review by PinkMoon25 — CHANGES_REQUESTED** [review]
Hi @lengushuru , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Linter checks :heavy_check_mark: 
Gitflow :heavy_check_mark: 
Removed boilerplate :heavy_check_mark: 
Calcaulator design :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @lengushuru 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewers. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**